src/components/shared/translations.js

```diff
@@ -5,6 +5,7 @@ const de = {
   errorMandatoryFieldsIntro: 'Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: ',
   errorMandatoryFieldsOutro: '. Erst dann können Sie Ihre Angaben speichern.',
   errorMandatoryFieldsDefault: 'Bitte füllen Sie alle Pflichtfelder korrekt aus. Erst dann können Sie Ihre Angaben speichern.',
+  placeholder: "Suche"
```

Comment:
I'd suggest to specify both the key and the value. "Suche" is too vague, and so is "placeholder".

```suggestion
  placeholderAutoSuggest: "Suchbegriff..."
```

Reply:
Done also in https://github.com/demos-europe/demosplan-ui/pull/396/commits/c176fd60bb45475146988f0460f8d82c0a3cc27d